src/genpy/message.py

```diff
@@ -48,6 +48,11 @@
 from .base import is_simple
 from .rostime import Time, Duration, TVal
 
+try:
+    reload         # Python 2
```

Comment:
The comment needs to start two spaces after the function name to pass PEP8.

Reply:
Quoting from PEP8 (the emphasis is mine):
* An inline comment is a comment on the same line as a statement. Inline comments should be separated by __at least__ two spaces from the statement. They should start with a # and a single space.

Reply:
That is right. `flake8` is fine with additional spaces. Please avoid this kind of comment alignment anyway since it necessarily couples these lines in case of future changes.

Reply:
Please update the line anyway as suggested.

Reply:
Although python 2 will be officially EOL, there's still a long tail of platforms with longer support cycles that still have python2 installed on them that will need to be continued supporting. (just as an example Ubuntu Bionic is EOL in 2023)

Reply:
Ubuntu Bionic dos NOT have Python 2 installed:
* https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes

Reply:
> Ubuntu Bionic dos NOT have Python 2 installed

The fact that Bionic doesn't install a Python 2 interpreter by default is not relevant. The ROS 1 Debian packages do use Python 2.